net/routing/src/opnsense/service/templates/OPNsense/Quagga/ospfd.conf

```diff
@@ -0,0 +1,49 @@
+{% macro cline(directive, modelname) -%}{% if modelname %}
+  ip ospf {{ directive }} {{ modelname }}
+{% endif %}{%- endmacro %}
+!
+! Zebra configuration saved from vty
+!   2017/03/03 20:21:04
+!
+!
+!
+!
+{% if helpers.exists('OPNsense.quagga.ospf.interfaces.interface') %}
+{%   for interface in OPNsense.quagga.ospf.interfaces.interface %}
```

Comment:
@fabianfrz you can wrap it in helpers.toList() like https://github.com/opnsense/core/blob/5f7fa5900d6b71f57d38998c5c8bd334955ec864/src/opnsense/service/templates/OPNsense/IDS/rule-updater.config#L7 when a migration is provided the outer "if" could be removed as well, because the structure should be created after install (but there  are more similar cases in the rest of our code, it won't hurt)